endpointmanager/pkg/endpointmanager/postgresql/fhirendpointmetadatastore_test.go

```diff
@@ -254,5 +256,90 @@ func Test_PersistFHIREndpointMetadata(t *testing.T) {
 	if count != 1 {
 		t.Errorf("expected 1 update for endpointInfo1 in history table. Got %d.", count)
 	}
+}
+
+func Test_AvailabilityUsesMetadataRequestedVersion(t *testing.T) {
+	SetupStore()
+	teardown, _ := th.IntegrationDBTestSetup(t, store.DB)
+	defer teardown(t, store.DB)
+
+	var err error
+	ctx := context.Background()
+
+	// capability statement
+	path := filepath.Join("../../testdata", "cerner_capability_dstu2.json")
+	csJSON, err := ioutil.ReadFile(path)
+	if err != nil {
+		t.Error(err)
+	}
+	cs, err := capabilityparser.NewCapabilityStatement(csJSON)
+	if err != nil {
+		t.Error(err)
+	}
+
+	var endpointMetadata1 = &endpointmanager.FHIREndpointMetadata{
+		URL:               "example.com/FHIR/DSTU2/",
+		HTTPResponse:      200,
+		Errors:            "Example Error",
+		SMARTHTTPResponse: 0,
+		Availability:      1.0,
+		RequestedFhirVersion: "None"}
+
+	var endpointMetadata2 = &endpointmanager.FHIREndpointMetadata{
+		URL:               "other.example.com/FHIR/DSTU2/",
+		HTTPResponse:      404,
+		Errors:            "Example Error 2",
+		SMARTHTTPResponse: 0,
+		Availability:      0,
+		RequestedFhirVersion: "1.0.2"}
+
+	// endpointInfos
+	var endpointInfo1 = &endpointmanager.FHIREndpointInfo{
```

Comment:
It doesn't seem like endpointInfo1 and endpointInfo2 are really used since you could just use the URL from the metadata below